src/backend/api/reservations.js

```diff
@@ -0,0 +1,78 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+//select all
+router.get("/", async (request, response) => {
+  try {
+    const reservations = await knex("reservation").select("*");
+    response.json(reservations);
+  } catch (error) {
+    throw error;
+  }
+});
+module.exports = router;
+//add new reservation
+router.post("/", async (request, response) => {
+  try {
+    const addReservation = request.body;
+    const [reservationId] = await knex("reservation").insert({
+      number_of_guests: addReservation.number_of_guests || "Untitled",
+      meal_id: addReservation.description,
+      created_date: addReservation.created_date,
```

Comment:
I would remove this field from here.

Reply:
should I just delete the created_date!?

Reply:
or should I make a variable like let today = new Date().toISOString().slice(0, 10);
and then assign it to created_date: addReservation.today